frontend/src/pages/destructionlist/detail/components/DestructionListProcessZaakReviewModal/DestructionListProcessZaakReviewModal.tsx

```diff
@@ -192,10 +200,18 @@ export const DestructionListProcessZaakReviewModal: React.FC<
       },
 
       {
-        label: isArchiefactiedatumActive ? "archiefactiedatum" : undefined,
+        label:
+          isArchiefactiedatumActive &&
+          !getBewaartermijn(_formState.selectielijstklasse)
+            ? "archiefactiedatum"
+            : undefined,
         name: "archiefactiedatum",
         required: true,
-        type: isArchiefactiedatumActive ? "date" : "hidden",
+        type:
+          isArchiefactiedatumActive &&
+          !getBewaartermijn(_formState.selectielijstklasse)
```

Comment:
Is there a reason for not adding `&& getBewaartermijn(_formState.selectielijstklasse)` on line 155 instead so that you don't have to call it twice?